modules/packages/Sort.chpl

```diff
@@ -130,6 +142,67 @@ implemented with a compare method:
   // This will output: -1, 2, 3, -4
   writeln(Array);
 
+The .keyPart method
+~~~~~~~~~~~~~~~~~~~
+
+A ``keyPart(a, i)`` method returns *parts* of key value at a time. This
+interface supports radix sorting for variable length data types, such as
+strings. It accepts two arguments:
+
+ * ``a`` is the element being sorted
+ * ``i`` is the part number of the key requested, starting from 1
+
+A ``keyPart`` method should return a tuple consisting of *section* and a *part*.
+
+ * The *section* can be any signed integral type and should have the value `-1`,
+   `0`, or `1`. It indicates when the end of the ``a`` has been reached
+   and in that event how it should be sorted relative to other array elements.
+
+   ================ ====================================
+   Returned section Interpretation
+   ================ ====================================
+   ``-1``           no more key parts for ``a``,
```

Comment:
Since these are actually `<0`, `0` and `>0`, would you like to use the notation from the compare method above?

Reply:
I think it should document the more restrictive thing for now

Reply:
Works for me.